libportal/notification.c

```diff
@@ -180,6 +181,9 @@ call_done (GObject *source,
  * - default-action-target `v`: target parameter to send along when
  *     activating the default action.
  * - buttons `aa{sv}`: array of serialized buttons
+ * - desktop-file-id `s`: A desktop file id containing the .desktop suffix.
+ * - display-hint `as`: An array of display hints.
+ * - category `s`: A category for this notification.
```

Comment:
Could you include a link to valid hints and categories?

Reply:
done